### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by smonff**
I focused on reviewing the text only and fixing minor typo. I didn't installed anything (will do later). This is only a small part but I already send this so tell me if the methodology is fine.

- [x] Features
  - [x] Scope
  - [x] Structure
  - [x] Federation
- [ ] Administrator Documentation
  - [x] Setup Guides

Never used RST before, I educated myself a bit and hope everything is fine.

### CODE REVIEW

File: docs/installation/external_dependencies.rst [hunk 1]
```diff
@@ -37,15 +37,15 @@ Create the project database and user:
...
+Assuming you have :ref:`created your reel2bits user <create-reel2bits-user>`,
 you should now be able to open a postgresql shell:
 
 .. code-block:: shell
 
     sudo -u reel2bits -H psql
 
 Unless you give a superuser access to the database user, you should also
-enable some extensions on your database server, as those are required
+enable some extensions on your database server. Tose are required
```
Comment (rhaamo) [inline docs/installation/external_dependencies.rst]: missing a `h` ?

**Comment by rhaamo** [comment]
looks globally good thanks !

there is a `./serve.py` in docs to compile the docs, assuming you install sphinx with python3 if you want to test the `..important` thing

**Comment by smonff** [comment]
> good ! thanks !

Still more to come but I wanted to already push this.

**Comment by rhaamo** [comment]
no problem, I just needed to merge this one to merge frontv2

**Pull request merged**